### PR TITLE
Fixes to test suite

### PR DESCRIPTION
**Pull request by severinson**

### CODE REVIEW

File: internal/testsuite/eventwatcher/eventwatcher.go [hunk 1]
```diff
@@ -210,11 +209,10 @@ func GetFromIngresses(parent context.Context, C chan *api.EventMessage) error {
 }
 
 func getFromIngress(ctx context.Context, host string) error {
-	tokens := strings.Split(host, ".")
-	if len(tokens) == 0 {
-		return errors.Errorf("malformed hostname: %s", host)
+	ingressUrl := os.Getenv("ARMADA_EXECUTOR_INGRESS_URL")
```
Comment (kannon92) [inline internal/testsuite/eventwatcher/eventwatcher.go:212]: It seems like you are missing the case of a bad hostname from host parameter.  

Is that intentional?

Comment (severinson) [reply internal/testsuite/eventwatcher/eventwatcher.go:221]: What do you mean?

Comment (severinson) [reply internal/testsuite/eventwatcher/eventwatcher.go:221]: I think that's fine.

Comment (kannon92) [reply internal/testsuite/eventwatcher/eventwatcher.go:221]: Kk.  I didn't see that this was in test suite.  Yea I think that's fine.

**Review by kannon92 — APPROVED** [review]

**Review by d80tb7 — APPROVED** [review]

**Pull request merged**